Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -797,7 +823,8 @@ WarpXOpenPMDPlot::DumpToFile (ParticleContainer* pc,
         }
     }
 
-    m_Series->flush();
+    //m_Series->flush();
```

Comment:
```suggestion
```
You may want to erase this line or maybe replace it with a comment on why `m_Series->flush();` is no longer necessary.